Polish/przygoda/jedzenie/korwin/prezent/pistolet/pistolet.md

```diff
@@ -0,0 +1 @@
+Spod łóżka wyciągasz AK47. Korwin jest z Ciebie dumny, że masz taką broń i proponuje Ci wspólne poćwiczenie strzelania do tarczy na podwróku, przed domem.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.